### PR TITLE
feature: animation triggers and optional backdrop background color

### PR DESCRIPTION
**Pull request by gorda0**

### CODE REVIEW

File: example/src/App.js [hunk 1]
```diff
@@ -16,8 +16,13 @@ const Screen = {
...
 const snapPoints = [0, Screen.height / 2, '70%', '100%'];
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.sheetRef = React.createRef();
+  }
+
   onOpenBottomSheetHandler = (index) => {
-    this.refs.BottomSheet.snapTo(index);
+    this.sheetRef.current.snapTo(index);
```
Comment (nomi9995) [inline example/src/App.js:25]: here app is crashing. without `current` it is working good

File: src/index.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ const getInitialPosition = (snapPoint) => {
 class BottomPanel extends Component {
   constructor(props) {
     super(props);
-    this._deltaY = new Animated.Value(Screen.height);
+    this._deltaY = new Animated.Value(500);
```
Comment (nomi9995) [inline src/index.js:40]: can you elaborate on why you gave 500 static value?

File: src/index.js [hunk 3]
```diff
@@ -93,11 +101,16 @@ class BottomPanel extends Component {
       tipStyle,
       headerStyle,
       bodyStyle,
+      renderTip = true,
```
Comment (nomi9995) [inline src/index.js:104]: `isRoundBorderWithTipHeader` props is already available for `renderTip` therefore, no need to add `renderTip` props

File: src/index.js [hunk 4]
```diff
@@ -149,24 +168,26 @@ class BottomPanel extends Component {
...
+                      shadowRadius: 5,
+                      shadowOpacity: 0.4,
+                    }
+                  : {},
+                containerStyle,
+              ]
+            }
           >
-            {!isModal && isRoundBorderWithTipHeader && (
+            {!isModal && isRoundBorderWithTipHeader && renderTip && (
```
Comment (nomi9995) [inline src/index.js:190]: there is no need for `renderTip`

File: src/index.js [hunk 5]
```diff
@@ -149,24 +168,26 @@ class BottomPanel extends Component {
...
-                    shadowColor: '#000000',
-                    shadowOffset: { width: 0, height: 0 },
-                    shadowRadius: 5,
-                    shadowOpacity: 0.4,
-                  }
-                : {},
-              containerStyle,
-            ]}
+            style={
+              renderTip && [
```
Comment (nomi9995) [inline src/index.js:172]: there is no need for `renderTip`

**Review by nomi9995 — CHANGES_REQUESTED** [review]
Thanks. there is no need for `renderTip` because `isRoundBorderWithTipHeader` this props already exist for this purpose

File: src/index.js (hunk 2 above)
Comment (gorda0) [reply src/index.js]: Just a test mistake.

File: example/src/App.js (hunk 1 above)
Comment (gorda0) [reply example/src/App.js]: fixed!

**Pull request closed without merging**